skills/declarative/calendarSkill/dialogs/SearchEventsDialog/SearchEventsDialog.dialog

```diff
@@ -0,0 +1,341 @@
+{
+  "$kind": "Microsoft.AdaptiveDialog",
+  "$designer": {
+    "id": "tMMCfJ",
+    "name": "SearchEventsDialog"
+  },
+  "autoEndDialog": true,
+  "defaultResultProperty": "dialog.result",
+  "triggers": [
+    {
+      "$kind": "Microsoft.OnBeginDialog",
+      "$designer": {
+        "name": "BeginDialog",
+        "id": "UnSYCl"
+      },
+      "actions": [
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "SapBYc"
+          },
+          "name": "SearchEventIn"
+        },
+        {
+          "$kind": "Microsoft.EmitEvent",
```

Comment:
I don't think we should introduce a new pattern here with the event. We can achieve the same thing by adding in the resolveDateTimeDialog to the intent triggers on calendarSkillDialog, and passing in the title and personNameArray, and datetime from the resolve dialog into SearchEvents as dialog options. Then just add the resolveDateTime and setProperties steps into the prompt in searchEvents.

Reply:
Sure, removed this pattern and simplified the logic